### DISCUSSION TITLE
Can't add team permission to document

### DISCUSSION THREAD

**Discussion by ghost** [appwrite/appwrite#5212] [Q&A] [answered]
### 👟 Reproduction steps

1) Login using JWT on the server
2) Try to add a document to the database with extra permissions.

Example:
```
      await database.createDocument(
        DB_ID,
        CLIENT_CL_ID,
        ID.unique(),
        req.body,
        [Permission.write(Role.team(ADMIN_TEAM_ID))]
      );
```

Error: 

```
AppwriteException [Error]: Permissions must be one of: (any, users, user:640ad0bab08912c0ecfb, user:640ad0bab08912c0ecfb/verified, users/verified, team:640acd669720e43a6583, member:640ad0d847c6e7b2da85, team:640acd669720e43a6583/client)
```

### 👍 Expected behavior

I want to add the doc to the database. The condition is that document access will be granted to the user and the team admin. But when I add the team permission, an appwrite exception is displayed.

### 👎 Actual Behavior

The document should be created without any errors. And the user and team admin should only have access to the document.

### 🎲 Appwrite version

Version 1.2.x

### 💻 Operating system

Windows

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment, marked as answer]
@vlohar08, thanks for creating this issue! 🙏🏼 This is actually the expected behavior. From our [docs](https://appwrite.io/docs/permissions):

> A project user can only grant a resource with permissions that they own. For example, if a user is trying to share a document with a team that they are not a member of, they will encounter a 401 not authorized error. If your app needs users to grant access to teams they're not a member of, you can create Appwrite Functions with a [Server SDK](https://appwrite.io/docs/getting-started-for-server) to achieve this functionality.

In your case, since you have server-side code, you can use an API key instead of the JWT to create the document.
